README.md

```diff
@@ -2,9 +2,31 @@
 
 ![QuickPizza screenshot](./docs/images/quickpizza-screenshot.png)
 
+- [QuickPizza](#quickpizza)
```

Comment:
Adding this small table of contents to show various sections of the readme easily